docs/analyst_guide.md

```diff
@@ -235,6 +236,27 @@ Default [Postgres function](https://www.postgresql.org/docs/14/functions-string.
 
 **Restrictions:** In untrusted mode, only `start = 1` is allowed.
 
+## Type casts
+
+When selecting or generalizing columns of personal tables, the following type conversions are allowed.
+
+All numeric types may be converted to other numeric types.
+Numeric types are: `smallint`, `integer`, `bigint`, `float`, `double`, and `numeric` (`decimal`).
+When converting from a real type to an integer type, implicit rounding occurs (away from zero).
```

Comment:
We should underline here that this counts as a generalization expression (i.e. another generalization can't occur at the same time).